types/jquery-editable-select/tslint.json

```diff
@@ -0,0 +1,6 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "ban-types": false
```

Comment:
Please fix this lint error, or add `// tslint:disable-line ban-types` if you think you have a genuine use of one of the banned types.